docs/imaging/sync.rst

```diff
@@ -4,31 +4,117 @@
 
-.. figure:: /_static/imaging/wavesurfer_sync-01.jpg
+.. figure:: /_static/imaging/sync/wavesurfer_sync-01.jpg
    :alt: Synchronisation of parallel data streams
 
+Wavesurfer stores its data in an ``hdf5`` container. A reader for these containers is provided under `/helpers/mini2p_wavesurfer.py <https://github.com/kavli-ntnu/dj-moser-imaging/blob/master/helpers/mini2p_wavesurfer.py>`_. 
+
 Scanimage sync
 ---------------------
 
-Data can be ``pre-synced`` with scanimage. That means that either scanimage is triggering a - for example - tracking camera for every frame it acquires or timestamp information is recorded at imaging clock rate on one of 4 ``AUX`` inputs. The timing info for every frame and also an array of time stamps at imaging clock precision (`AUX 0 to 3`) get saved in the header of every page of the scanimage tif file. 
\ No newline at end of file
+The OpenMiniscope system implements one particular special case of the more general synchronisation scheme outlined above. Data can be *pre-synchronised* with the `ScanImage <http://scanimage.vidriotechnologies.com/display/SIH/ScanImage+Home>`_ software. In this case, instead of each instrument operating at its own sampling frequency determined by its own independent clock, each frame of acquisition is triggered by a signal sent from a single "master" source, which is the device controlling the two-photon imaging acquisition. Each time that ScanImage triggers a frame of two-photon imaging data, it can send a trigger signal to other instruments, causing them to acquire a frame of data simultaneously, or at least with a known, constant, offset. 
+
+In this case, the timestamp data for two-photon imaging frames is identical to the timestamp data for tracking frames.
+
+Note that although this method is superficially much simpler than the more general case, it has its own complexities, especially where multi-plane two-photon imaging is in use. In general, the design of the OpenMiniscope system is to send one trigger to external instruments on each *volume* acquisition, rather than on each *plane* acquisition, and consequently, there may be an integer multiple between the number of plane acquisitions and the nnumber of tracking frames. 
+
+For instruments where the acquisition rate of the two-photon imaging frames is too slow, the system supports acquiring a limited number of data streams at the acquisition rate of the two-photon photodetectors. In this case, a much higher temporal resolution can be saved as an "imaging frame", generating a TIF image (essentially, an overengineered array of data) alongisde the imaging data.
+    
```

Comment:
" In this case, a much higher temporal resolution can be saved as an "imaging frame", generating a TIF image (essentially, an overengineered array of data) alongisde the imaging data." -> No, this doesn't increase the temporal resolution at all (since timestamps within (!) a frame acquisition cannot be resolved).

Better to refer to the AUX and I2C signals here which generate timestamps at scanimage's internal clock speed and are therefore arbitrarily precise compared to signals locked to the imaging acquisition itself.